compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -408,7 +410,7 @@ pub fn mk_attr_name_value_str(
     val: Symbol,
     span: Span,
 ) -> Attribute {
-    let lit = LitKind::Str(val, StrStyle::Cooked).to_token_lit();
+    let lit = LitKind::Str(val, StrStyle::Cooked).synthesize_token_lit();
```

Comment:
Also suspicious for the same reason as `mk_name_value_item`.

Reply:
Removed.